lib/linguist/languages.yml

```diff
@@ -2003,6 +2003,7 @@ Hack:
   codemirror_mode: php
   codemirror_mime_type: application/x-httpd-php
   extensions:
+  - ".hack"
```

Comment:
would be great to have at least one `*.hack` file in the [samples section](https://github.com/github/linguist/tree/master/samples/Hack) if this is the standard extension. Instead of linking to external sites, you could then add directly a lightshow example for these sample files.

Reply:
Good idea, I didn't know there was a samples section in the repository. I'll add some.